src/main/java/world/bentobox/bentobox/database/objects/Players.java

```diff
@@ -34,6 +36,13 @@
     @Expose
     private Map<String, Integer> deaths = new HashMap<>();
 
+    /**
+     * This variable stores set of worlds where user inventory is quarantined.
+     */
+    @Expose
+    private Set<String> quarantineWorld = new HashSet<>();
```

Comment:
I don't agree with the name. That does not sound right. I'd rather like something like "pendingIslandsReset" (this name is not perfect as well). We need to think of a clear and concise name.

Reply:
That's up to you.

Reply:
Okay, call this set `pendingKicks` (and update its javadoc, and add an `@since 1.3.0`)